src/main/java/org/dita/dost/writer/KeyrefPaser.java

```diff
@@ -536,6 +537,19 @@ private Attributes processElement(final Attributes atts) {
                                 String topicId = null;
                                 if (relativeTarget.getFragment() == null && !"".equals(elementId)) {
                                     topicId = getFirstTopicId(topicFile);
+                                    if (topicId == null) {
+                                        String directHref = keyDef.element.attribute(ATTRIBUTE_NAME_HREF);
+                                        if(directHref != null && !directHref.equals(href.toString())) {
```

Comment:
Add space after `if`. Amend the previous style fix commit and force push instead of creating a new commit.